dotnetv3/dynamodb/scenarios/DynamoDB_Basics/DynamoDB_Basics_ScenarioTests/DynamoDbMethodsTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace DynamoDB_Basics_Scenario.Tests
+{
+    using Xunit;
+    using DynamoDB_Basics_Scenario;
+    using System.Threading.Tasks;
+    using Amazon.DynamoDBv2;
+
+    [TestCaseOrderer("OrchestrationService.Project.Orderers.PriorityOrderer", "OrchestrationService.Project")]
+    public class DynamoDbMethodsTests
+    {
+        readonly AmazonDynamoDBClient client = new();
+        readonly string tableName = "movie_table";
+        readonly string movieFileName = "moviedata.json";
+        readonly string testMovieFileName = "testmoviedata.json";
+
+        [Fact, TestPriority(1)]
+        public async Task CreateMovieTableAsyncTest()
+        {
+            var success = await DynamoDbMethods.CreateMovieTableAsync(client, tableName);
+
+            Assert.True(success, "Failed to create table.");
+        }
+
+        [Fact, TestPriority(2)]
+        public async Task PutItemAsyncTest()
+        {
+            var newMovie = new Movie
+            {
+                Year = 1959,
+                Title = "North by Northwest",
+            };
+
+            var success = await DynamoDbMethods.PutItemAsync(client, newMovie, tableName);
+            Assert.True(success, "Couldn't add the movie.");
+        }
+
+        [Fact, TestPriority(3)]
+        public void UpdateItemAsyncTest()
+        {
+            Assert.True(false, "This test needs an implementation");
```

Comment:
Looks like there's still work to be done here.

Reply:
Added the missing test and it passes.